source/debian/Demo_User_Guides/Chromium_Browser.rst

```diff
@@ -38,39 +39,34 @@ Launching Chromium Browser
 
    ``--no-sandbox`` option is required only when running as root user.
 
-To launch the Chromium browser:
-
-Assuming you are logged in as the root user.
+The following is assuming that you are logged in as the root user. The syntax of the command is as follows:
 
 .. code-block:: console
 
     $ /usr/bin/chromium [url] --use-gl=angle --ozone-platform=wayland --no-sandbox [options]
 
-e.g.
+For example, the following command opens www.ti.com in a windowed browser on the Weston desktop.
 
 .. code-block:: console
 
     $ chromium https://www.ti.com --use-gl=angle --ozone-platform=wayland --no-sandbox
 
-Will open www.ti.com in a windowed browser on the Weston desktop.
+The following command opens the aquarium 3D benchmark in a fullscreen window on the Weston desktop. This benchmark
```

Comment:
Nitpick,

Better to keep "**provided you have network connectivity to the internet from the TI platform**" phrase intact.